python_modules/libraries/dagster-dbt/dagster_dbt_tests/cli/test_resources_v2.py

```diff
@@ -25,6 +26,58 @@ def test_dbt_cli(global_config: List[str], command: str) -> None:
     assert dbt_cli_task.process.returncode == 0
 
 
+def test_dbt_cli_subsetted_execution(mocker: MockerFixture) -> None:
+    mock_context = mocker.MagicMock()
+    mock_selected_output_names = ["least_caloric", "sort_by_calories"]
+
+    type(mock_context.op).tags = mocker.PropertyMock(return_value={})
```

Comment:
would be nice to test this in a context where select / exclude were set by the user, just to verify that those don't override the subselection (not that I think they do)